### PR TITLE
making the selected theme bg color lighter

### PR DESCRIPTION
**Pull request by emre2038**
## Description
calculating the background color of the selected category 

## How is it tested automatically?
*

## Checklist

- [ ]  🚀 is the code ready to be merged and go live?
- [ ]  🛠 does it work (build) locally
- [ ] 👩‍🎨 does the design match the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [ ]  📰 good title
- [ ]  📝good description
- [ ]  🔖 issue linked
- [ ]  📖 changelog filled out
- [ ] commit messages are meaningful

### Code Quality

- [ ]  🚧 no commented out code
- [ ]  🖨 no unnecessary logging
- [ ]  🎱 no magic numbers

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

File: src/js/utils.js [hunk 1]
```diff
@@ -482,18 +482,39 @@ export function trimValue(val) {
...
+        a = alpha;
+    } else {
+        a = '01';
     }
+    // multiply before convert to HEX
+    a = ((a * 255) | 1 << 8).toString(16).slice(1)
+    hex = hex + a;
 
-    return c;
+    return hex;
```
Comment (michaelglenister) [inline src/js/utils.js:519]: Is there a reason this returns the value without the # symbol, could we rather do something like 

`return "#" + hex;`

**Review by michaelglenister — COMMENTED** [review]
Possible one small change, otherwise it looks good

File: src/js/utils.js (hunk 1 above)
Comment (emre2038) [reply src/js/utils.js:519]: done 👍

**Review by michaelglenister — APPROVED** [review]
